BrowserKit/Sources/Common/Utilities/FontStyle.swift

```diff
@@ -0,0 +1,56 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import UIKit
+
+public struct Styling {
+    private let textStyle: UIFont.TextStyle
+    private let size: CGFloat
+    private let weight: UIFont.Weight
+
+    init(for textStyle: UIFont.TextStyle, size: CGFloat, weight: UIFont.Weight) {
+        self.textStyle = textStyle
+        self.size = size
+        self.weight = weight
+    }
+
+    public func scaledFont() -> UIFont {
+        return DefaultDynamicFontHelper.preferredFont(withTextStyle: textStyle, size: size, weight: weight)
+    }
+
+    public func systemFont() -> UIFont {
+        UIFont.systemFont(ofSize: size, weight: weight)
+    }
+}
+
+public struct FontStyle {
```

Comment:
This is aa possibly ignorable nit, but I would probs call this `FXFontStyles` to match `FXColours`.

Reply:
I updated to `FXFontStylez`, so we shall see 😄

Reply:
😆😆😆 I was looking at the code and was curious why there was a Z at the end there. Now I know why, but I'll be the not nice boring person saying maybe we shouldn't put a Z there hahaha 😅